src/strings/PowerFxResources.en-US.resx

```diff
@@ -1022,6 +1022,44 @@
   <data name="AboutISOWeekNum_date" xml:space="preserve">
     <value>A date value for which the ISO week number will be calculated.</value>
   </data>
+  <data name="AboutEDate" xml:space="preserve">
+    <value>Returns the date adjusted by a number of months.  The date portion of the result remains the same but is capped by the number of days in the new month.</value>
```

Comment:
>date portion [](http://example.com/codeflow?start=65&length=12)

'The *date* portion...' or 'The *day* portion?

Reply:
Done.